src/current/v25.1/understand-hotspots.md

```diff
@@ -0,0 +1,308 @@
+
+#### Lookback Hotspots
+
+**Synonyms:** index scan hotspot, moving read hotspot
+
+A _lookback hotspot_ is a type of index hotspot in which a moving subset of an index is being read in a way that burdens the cluster.
+
+For example, consider querying a `posts` table for the most recently created post: 
+
+~~~ sql
+CREATE TABLE posts(
+   id UUID,
+   created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+   INDEX (created_at) USING HASH
+);
+
+SELECT MAX(created_at) FROM posts GROUP BY created_at ORDER BY created_at LIMIT 1;
+~~~
+
+What makes lookback hotspots unique is that the cluster is not hot by write, but hot by read. Separately lookback hotspots also tend not to specify a key, which would evade systems using key requests to identify hotspots.
+
+#### Queuing Hotspot
+
+**Synonyms:** Outbox hotspot
+
+A _queuing hotspot_ is a type of index hotspot that occurs when an workload treats CockroachDB like a distributed queue. This can happen if you implement the [Outbox microservice pattern]({% link {{ page.version.version }}/cdc-queries.md %}#queries-and-the-outbox-pattern).
+
+Queues, such as logs, generally require data to be ordered by write, which necessitates indexing in a way that is likely to create a hotspot. An outbox where data is deleted as it is read has an additional problem: it tends to accumulate an ordered set of [garbage data]({% link {{ page.version.version }}/operational-faqs.md %}#why-is-my-disk-usage-not-decreasing-after-deleting-data) behind the live data. Since the system cannot determine whether any live rows exist within the garbage data, what appears to be a small table scan to the user can actually result in an unexpectedly intensive scan on the garbage data.
+
+<img src="{{ 'images/v25.1/hotspots-figure-5.png' | relative_url }}" alt="Outbox hotspot example" style="border:1px solid #eee;max-width:100%" />
+
+To mitigate this issue, it is advisable to use [Change Data Capture (CDC)]({% link {{ page.version.version }}/cdc-queries.md %}) to ensure subscription to updates instead of using Outbox tables. If using CDC is not possible, sharding the index that the outbox uses for ordering can reduce the likelihood of a hotspot within the cluster.
+
+### Row Hotspot
+
+**Synonyms:** hot key, hot row, point hotspot
+
+A _row hotspot_ is a hotspot where an individual point in the keyspace becomes throughput limiting, often due to high activity on a single row. The word "point" illustrates the fact that the location that is hot is indivisible, and therefore cannot be [split]({% link {{ page.version.version }}/load-based-splitting.md %}). A classic example is a social media application with a high volume of activity on certain users.
```

Comment:
Since there isn't reference to point hotspots before this point, might be worth scrapping the reference:

`Since rows are inherently indivisible, a hotspot on a row cannot be split...`